sources/packages/backend/apps/api/src/services/application-bulk-withdrawal/application-bulk-withdrawal-import-business-validation.models.ts

```diff
@@ -0,0 +1,310 @@
+  }
+
+  /**
+   * Context type.
+   */
+  type: ValidationContextTypes;
+  /**
+   * Unique identifier of the validation error.
+   */
+  typeCode: ApplicationWithdrawalValidationWarnings;
+}
+
+export class ApplicationBulkWithdrawalImportBusinessValidationModel {
+  /**
+   * Data record type.
+   */
+  @ValidateIf(
+    (object: ApplicationBulkWithdrawalImportBusinessValidationModel) =>
+      object.applicationFound,
+  )
+  @Equals(RecordType.ApplicationBulkWithdrawalDataRecordType, {
+    message: `${DataTextHeaders.recordType} must be valid.`,
+  })
+  recordType: RecordType;
+  /**
+   * SIN.
+   */
+  @ValidateIf(
+    (object: ApplicationBulkWithdrawalImportBusinessValidationModel) =>
+      object.applicationFound,
+  )
+  @IsNumberString(
+    { no_symbols: true },
+    {
+      message: `${DataTextHeaders.sin} must be a valid number.`,
+    },
+  )
+  @Length(SIN_NUMBER_LENGTH, SIN_NUMBER_LENGTH, {
+    message: `${DataTextHeaders.sin} must be of length ${SIN_NUMBER_LENGTH}.`,
+  })
+  sin: string;
+  /**
+   * Application Number.
+   */
+  @ValidateIf(
+    (object: ApplicationBulkWithdrawalImportBusinessValidationModel) =>
+      object.applicationFound,
+  )
+  @IsNumberString(
+    { no_symbols: true },
+    {
+      message: `${DataTextHeaders.applicationNumber} must be a valid number.`,
+    },
+  )
+  @Length(APPLICATION_NUMBER_LENGTH, APPLICATION_NUMBER_LENGTH, {
+    message: `${DataTextHeaders.applicationNumber} must be of length ${APPLICATION_NUMBER_LENGTH}.`,
+  })
+  applicationNumber: string;
+  /**
+   * Withdrawal Date.
+   */
+  @ValidateIf(
+    (object: ApplicationBulkWithdrawalImportBusinessValidationModel) =>
+      object.applicationFound,
+  )
+  @IsDate({
+    message: `${DataTextHeaders.withdrawalDate} must be a valid withdrawal date.`,
+  })
+  withdrawalDate: Date;
+  /**
+   * Application found.
+   */
+  @IsBoolean({
+    message: `${userFriendlyNames.applicationFound} must be a boolean value.`,
+  })
+  @IsIn([true], {
+    message: `${userFriendlyNames.application} is not present in SIMS and is a part of the SFAS system.`,
+    context: ValidationContext.CreateWarning(
+      ApplicationWithdrawalValidationWarnings.ApplicationNotFound,
+    ),
+  })
+  applicationFound: boolean;
+  /**
+   * Application belongs to institution.
+   */
+  @ValidateIf(
+    (object: ApplicationBulkWithdrawalImportBusinessValidationModel) =>
+      object.applicationFound,
+  )
+  @IsBoolean({
+    message: `${userFriendlyNames.applicationBelongsToInstitution}  must be a boolean value.`,
+  })
+  @IsIn([true], {
+    message: `${userFriendlyNames.application} does not belong to this institution.`,
+  })
+  applicationBelongsToInstitution: boolean;
+  /**
+   * Valid SIN.
+   */
+  @ValidateIf(
+    (object: ApplicationBulkWithdrawalImportBusinessValidationModel) =>
+      object.applicationFound,
+  )
+  @IsBoolean({
+    message: `${userFriendlyNames.validSIN} must be a boolean value.`,
+  })
+  validSIN: boolean;
```

Comment:
I would assume that `valid` here means the provided one matches the student one, right?
If it is right, can we have it renamed to `studentSINMactch` or similar?